Classes/common/CDTMutableDocumentRevision.m

```diff
@@ -50,25 +49,14 @@ - (instancetype)initWithDocumentId:(NSString*) documentId
     
     if (self){
         _docId = documentId;
-        _private_body = body ? body : [NSMutableDictionary dictionary];
-        _private_attachments = attachments ? attachments : [NSMutableDictionary dictionary];
+        _body = body ? body : [NSMutableDictionary dictionary];
+        _attachments = attachments ? attachments : [NSMutableDictionary dictionary];
         _sourceRevId = sourceRevId;
     }
     
     return self;
 }
 
-- (void)setBody:(NSDictionary *)body { self.private_body = [body mutableCopy]; }
-
-- (NSMutableDictionary *)body { return self.private_body; }
-
-- (NSMutableDictionary *)attachments { return self.private_attachments; }
-
-- (void)setAttachments:(NSMutableDictionary *)attachments
-{
-    self.private_attachments = [attachments mutableCopy];
```

Comment:
I have't fully thought through this suggestion, but: Can we make the `attachments` an `NSDictionary` instead of a mutable one?  Would that still allow us to make a `copy` and still keep the CDTMutableDocumentRevision a mutable thing?